src/components/review/write/PreviewImageList.tsx

```diff
@@ -0,0 +1,110 @@
+import Image from 'next/image';
+import DeleteIcon from 'public/assets/ic_delete_round.svg';
+import PlusIcon from 'public/assets/ic_img_plus.svg';
+import React from 'react';
+import shortId from 'shortid';
+import styled from 'styled-components';
+import { theme } from 'styles/theme';
+import { ReviewImage } from 'types/review';
+
+interface PreviewImageListProps {
+  reviewImageList: ReviewImage[];
+  handleImageUpload: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  handleImageDelete: (index: number) => void;
+  changeMainImage: (index: number) => void;
+}
+
+function PreviewImageList(props: PreviewImageListProps) {
+  const { reviewImageList, handleImageUpload, handleImageDelete, changeMainImage } = props;
+
+  const MAX_NUM = 9;
+
+  const imageList = [...reviewImageList];
+  imageList.splice(0, 1);
+
+  const emptyList =
+    imageList.length === MAX_NUM ? [] : new Array<null>(MAX_NUM - imageList.length - 1).fill(null);
+
+  return reviewImageList.length > 0 ? (
+    <StyledRoot>
+      {imageList.map((image, index) => (
+        <StyledImageCard key={shortId.generate()}>
+          <Image
+            src={image.preview ? image.preview : ''}
+            width={75}
+            height={75}
+            alt={`review${index}`}
+            onClick={() => changeMainImage(index + 1)}
+          />
+          <DeleteIcon onClick={() => handleImageDelete(index + 1)} />
+        </StyledImageCard>
+      ))}
+      {imageList.length !== MAX_NUM && (
+        <>
+          <StyledEmptyCard>
+            <PlusIcon />
+          </StyledEmptyCard>
+          <StyledInput type="file" accept="image/*" multiple onChange={handleImageUpload} />
+        </>
+      )}
+      {emptyList.map(() => (
+        <StyledEmptyCard key={shortId.generate()} />
+      ))}
+    </StyledRoot>
+  ) : (
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    <></>
+  );
+}
```

Comment:
```suggestion
  if (!reviewImageList.length) return null;

  return (
    <StyledRoot>
      {imageList.map((image, index) => (
        <StyledImageCard key={shortId.generate()}>
          <Image
            src={image.preview ? image.preview : ''}
            width={75}
            height={75}
            alt={`review${index}`}
            onClick={() => changeMainImage(index + 1)}
          />
          <DeleteIcon onClick={() => handleImageDelete(index + 1)} />
        </StyledImageCard>
      ))}
      {imageList.length !== MAX_NUM && (
        <>
          <StyledEmptyCard>
            <PlusIcon />
          </StyledEmptyCard>
          <StyledInput
            type="file"
            accept="image/*"
            multiple
            onChange={(e) => handleImageUpload(e, reviewImageList.length)}
          />
        </>
      )}
      {emptyList.map(() => (
        <StyledEmptyCard key={shortId.generate()} />
      ))}
    </StyledRoot>
  );
```

`reviewImageList`의 길이가 0인지를 미리 검사해서 null을 리턴하면 삼항연산자의 요소가 길어지는 것과 불필요한 fragment 태그의 반환을 막을 수 있을 것 같아요! 이 방법은 어떨까요

Reply:
null을 반환하면 warning 메세지 같은게 떠서 빈 컴포넌트를 반환하는 형식으로 짰었어요! 이 구조로 바꿔보도록 하겠습니다~~